svf-llvm/lib/LLVMUtil.cpp

```diff
@@ -1030,22 +1030,33 @@ const std::string SVFValue::toString() const
 {
     std::string str;
     llvm::raw_string_ostream rawstr(str);
-    if (const SVF::SVFFunction* fun = SVFUtil::dyn_cast<SVFFunction>(this))
-    {
-        rawstr << "Function: " << fun->getName() << " ";
-    }
-    else if (const SVFBasicBlock* bb = SVFUtil::dyn_cast<SVFBasicBlock>(this))
+
+    if (LLVMModuleSet::getLLVMModuleSet()->getLLVMValue(this) == nullptr)
     {
-        rawstr << "BasicBlock: " << bb->getName() << " ";
+        assert((SVFUtil::isa<SVFInstruction>(this) ||
+                SVFUtil::isa<SVFBasicBlock>(this)) && "this is not a manually created call inst by ExtAPI!");
```

Comment:
assert((SVFUtil::isa<SVFCallInst>(this) || SVFUtil::isa<SVFArgument>(this) ||
                SVFUtil::isa<SVFBasicBlock>(this)) && "Manually created call inst, argument and BasicBlock by ExtAP do not have LLVM valueI!");